package.json

```diff
@@ -132,7 +133,8 @@
     "lint:js": "eslint --ext .js ./",
     "lint:markdown": "markdownlint --config .markdownlint.json --ignore ./node_modules .",
     "profile": "webpack --progress=profile --json > logs/stats.json",
-    "readme": "webpack --mode=production --stats errors-warnings"
+    "readme": "webpack --mode=production --stats errors-warnings",
+    "vue-i18n-extract-translations": "vue-i18n-extract-translations -v './UI/src/**/*.?(js|vue)' -l './UI/src/locales/' --def-locale 'en' --key 'i18n' --keep-unused"
```

Comment:
maybe `i18n-extract` would be good? the command that's there now is almost as hard to remember as the command it replaces...

Reply:
Removed in favor of 'utils/devel/extract-vue-template-translations.sh' which keeps JSON & PO synced.